include/linkdefs.h

```diff
@@ -56,6 +56,8 @@ enum RPNCommand {
 	RPN_BANK_SELF		= 0x52,
 	RPN_SIZEOF_SECT		= 0x53,
 	RPN_STARTOF_SECT	= 0x54,
+	RPN_SIZEOF_SECTTYPE     = 0x55,
```

Comment:
Nitpick, but `SEG` is more consistent with... other environments. But maybe we'll update the terminology wholesale when implementing #524 instead?

Reply:
"seg" and "segment" are not yet terms that we use anywhere.

Reply:
I think it'll be worth switching the terminology then.

Reply:
Can that be a separate issue? "Refer to 'segments' not 'sections' everywhere"?

Reply:
Yeah, we'll do that when introducing HDFs.